### PR TITLE
Rsu roof area distribution

### PR DESCRIPTION
**Pull request by j3r3m1**
New dataset and tests for roof distrib indicator

### CODE REVIEW

File: geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy [hunk 1]
```diff
@@ -377,3 +377,142 @@ static IProcess rsuProjectedFacadeAreaDistribution() {
...
     )}
+
+/**
+ * Script to compute the distribution of roof (vertical, horizontal and tilted) area within a RSU per vertical layer.
+ * Note that the method used is based on the assumption that all buildings have gable roofs. Since we do not know
+ * what is the direction of the gable, we also consider that buildings are square in order to limit the potential
+ * calculation error (for example considering that the gable is always oriented parallel to the larger side of the
+ * building.
+ *
+ * @return A database table name.
```
Comment (SPalominos) [inline geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: Input description is missing

Comment (j3r3m1) [reply geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: You are right, I have added it on the same branch. Should I create a new branch to update the inputs and returns of each of the geoindicators process ?

Comment (SPalominos) [reply geoindicators/src/main/groovy/org/orbisgis/RsuIndicators.groovy]: As you want, you can do it in a new branch later.

**Comment by SPalominos** [comment]
@j3r3m1 ok for merge ?

**Comment by j3r3m1** [comment]
Yes definitely. I never know who is supposed to merge, I thought that I could not do that since I was the person who proposed the pull request.

**Comment by SPalominos** [comment]
You can merge your own PR if you have requested at least one review and if the reviewer(s) approved your PR.

**Pull request merged**